geodatasets/lib.py

```diff
@@ -118,6 +120,30 @@ def query_name(self, name: str) -> Dataset:
 
         raise ValueError(f"No matching item found for the query '{name}'.")
 
+    def filter_by_geometry(self, geometry_type: str) -> dict[str, Dataset]:
+        """Return dictionary of :class:`Dataset` objects filtered by geometry type
+
+        Returns dictionary of :class:`Dataset` objects from the :class:`Bunch`
+        filtered by `geometry_type` if found in :enum:`GeometryType`
```

Comment:
I think that the only cases we want to allow are "Point", "LineString", "Polygon", and potentially "Mixed" but we don't have any dataset of that kind now.

Reply:
sure, _had defined an `enum: geometry_types` in `geodatasets` and later switched to `shapely`_ .. all good 👍🏻